client/my-sites/domains/domain-management/list/bulk-site-domains.tsx

```diff
@@ -82,6 +82,10 @@ export default function BulkSiteDomains( props: BulkSiteDomainsProps ) {
 	const [ changeSiteAddressSourceDomain, setChangeSiteAddressSourceDomain ] =
 		useState< ResponseDomain | null >( null );
 
+	// If a site contains more than one domain that is not a subdomain.
+	const showManageAllDomainsCTA =
+		( data?.domains ?? [] ).filter( ( domain ) => ! domain.is_subdomain ).length > 1;
```

Comment:
@yansern Could you take another pass at this? The Manage All Domains button should show if there are domains on any other site.

Reply:
Just to verify, do you mean this?
 - If a site has 1 domain (excluding subdomains), it shouldn't show.
 - If a site has 2 or more domains (excluding subdomains), it should show.

Reply:
I meant this
* Manage All Domains doesn't show up if the user only has 1 site.
* Manage All Domains doesn't show up if the user has more than 1 site but only 1 domain.

Maybe the second one should be
* Manage All Domains doesn't show up if the user has more than 1 site but only 1 domain across those sites.

The Manage All Domains screen only makes sense for cases where users have more than one domains on more than one site.

Reply:
Okay gotcha, let me try again.

Reply:
Followed up here. https://github.com/Automattic/wp-calypso/pull/85414